app/src/hooks/useAudioStreamingOrchestrator.ts

```
@@ -45,13 +45,25 @@ export const useAudioStreamingOrchestrator = ({
+    // Force OMI stream codec to Opus:
+    // - replace existing codec query value, or
+    // - append codec=opus if missing.
+    if (/[?&]codec=/i.test(url)) {
+      url = url.replace(/([?&])codec=[^&]*/i, '$1codec=opus');
+    } else {
       const sep = url.includes('?') ? '&' : '?';
       url = url + sep + 'codec=opus';
     }
+    const isAdvanced = settings.jwtToken && settings.isAuthenticated;
 
     const isAdvanced = settings.jwtToken && settings.isAuthenticated;
```

Biome (2.4.4)

[error] Shouldn't redeclare 'isAdvanced'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)